<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordWaypointsWindow</class>
 <widget class="QDialog" name="RecordWaypointsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="startRecordButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Record</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopRecordButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Record</string>
   </property>
  </widget>
  <widget class="QListView" name="listWaypoints">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
